front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -72,14 +72,12 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
 
   const inputTerm = phase.attributes.input_term;
 
-  const sharedProps = {
-    className: participationMethod,
+  const sharedProps: WithFiltersProps = {
     ideaQueryParameters,
     onUpdateQuery: updateSearchParams,
     projectId,
     phaseId: phase.id,
     showViewToggle: true,
-    invisbleTitleMessage: messages.a11y_titleInputsPhase,
     defaultView: phase.attributes.presentation_mode,
```

Comment:
Indeed - didn't feel necessary for the "WithFilters" component. Though still necessary for the "WithoutFilters" I think, since that one doesn't have a title.